pdl_schema.md

```diff
@@ -0,0 +1,1106 @@
+---
+layout: guide
+title: PDL Schema
+permalink: /pdl_schema
+redirect_from: /pdl_syntax
+excerpt: Rest.li PDL Schema.
```

Comment:
This excerpt is used as the description text in the search functionality, so keep that in mind here. Something like "Documentation of pegasus schemas and specification of the PDL syntax."